spring-cloud-dataflow-docs/src/main/asciidoc/index.adoc

```diff
@@ -24,13 +24,13 @@ Sabby Anandan; Marius Bogoevici; Eric Bottard; Mark Fisher; Ilayaperumal Gopinat
 
 :dataflow-asciidoc: https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/master/spring-cloud-dataflow-docs/src/main/asciidoc
 
-:docker-http-source-rabbit-version: 1.3.1.RELEASE
-:docker-time-source-rabbit-version: 1.3.1.RELEASE
-:docker-log-sink-rabbit-version: 1.3.1.RELEASE
-:docker-log-sink-kafka-version: 1.3.1.RELEASE
-:docker-http-source-kafka-version: 1.3.1.RELEASE
-:docker-time-source-kafka-version: 1.3.1.RELEASE
-:docker-timestamp-task-version: 1.3.1.RELEASE
+:docker-http-source-rabbit-version: 2.1.0.RELEASE
+:docker-time-source-rabbit-version: 2.1.0.RELEASE
+:docker-log-sink-rabbit-version: 2.1.0.RELEASE
+:docker-log-sink-kafka-version: 2.1.0.RELEASE
+:docker-http-source-kafka-version: 2.1.0.RELEASE
+:docker-time-source-kafka-version: 2.1.0.RELEASE
+:docker-timestamp-task-version: 2.0.1.RELEASE
```

Comment:
Sorry for the wrong info in the previous review. We don't have 2.0.1 yet, the latest is still at 2.0.0 for `timestamp`.

Reply:
done on merge